Resources/Prototypes/Entities/Clothing/Back/backpacks.yml

```diff
@@ -201,3 +201,16 @@
     equippedPrefix: holding
   - type: Storage
     capacity: 9999
+
+- type: entity
+  parent: ClothingBackpack
```

Comment:
inherit from clown backpack and update components accordingly to that

Reply:
also done